src/sync/index.js

```diff
@@ -369,7 +134,10 @@ async function sync(db){
       });
     });
   },
-  async function(){
+  async function() {
+    // Insert Oracle names and options
+    await updateOracleInfo(db, oraclesNeedInfoUpdate);
```

Comment:
I dunt think we should put insert after init sync is done, would like to populate entries into db asap

Reply:
done as discussed